radiomics/imageoperations.py

```diff
@@ -13,42 +13,64 @@
 logger = logging.getLogger(__name__)
 
 
-def getHistogram(binwidth, parameterValues):
-  """
+def getBinEdges(binwidth, parameterValues):
```

Comment:
> I didn't test, but looking at the code, it seems that the binning strategy just calculates the width of the bin: 

It looks like that to me too. Still raises the question on whether to enable this. Main issue remains that any of these functions calculate bin widths on the input range, meaning that the comparability of gray values may be limited...